canonical-web-and-design/snapcraft.io
# Update Python dependencies

## Done

Update our Python dependencies

## Issue / Card

Fixes #2935

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check the website works fine, especially tutorials and docs.
